libcontainer/cgroups/systemd/v2.go

```diff
@@ -292,6 +296,32 @@ func (m *unifiedManager) Apply(pid int) error {
 	if err := fs2.CreateCgroupPath(m.path, m.cgroups); err != nil {
 		return err
 	}
+
+	goodToChown := []string{"cgroup.procs", "cgroup.subtree_control", "cgroup.threads"}
+	member := func(haystack []string, needle string) bool {
+		for _, v := range haystack {
+			if v == needle {
+				return true
+			}
+		}
+		return false
```

Comment:
Perhaps it's easier to use `map[string]struct{}` here.

Or just be direct
```
if name == "cgroup.procs" || name == "cgroup.subtree_control" || name == ....
```

Reply:
OK, I'll use a map.